merfi/backends/rpm_sign.py

```diff
@@ -47,6 +47,8 @@ def sign(self):
         logger.info('Starting path collection, looking for files to sign')
         self.keyfile = self.parser.get('--keyfile', 'Release.gpg')
```

Comment:
I've updated this PR so that it only adds basic tests, and doesn't touch `--keyfile` yet. I'll handle the `--keyfile` work in another PR.